src/urls.json

```diff
@@ -1,4 +1,12 @@
 { 
```

Comment:
the settings.yaml.dist is empty, you could put a placeholder with
`run_timeout: 3600`

Reply:
But now `run_timeout` is no more used so I deleted it